specification/contosowidgetmanager/Contoso.WidgetManager/main.tsp

```diff
@@ -36,23 +38,24 @@ model Widget {
   sharedModel?: FakedSharedModel;
 }
 
+interface StandardOperations extends Azure.Core.ResourceOperations<Azure.Core.Traits.NoClientRequestId & Azure.Core.Traits.NoConditionalRequests & Azure.Core.Traits.NoRepeatableRequests, Azure.Core.Foundations.ErrorResponse> {}
 interface Widgets {
   @doc("Fetch a Widget by name.")
-  getWidget is ResourceRead<Widget>;
+  getWidget is StandardOperations.ResourceRead<Widget>;
```

Comment:
For my learning, is this considered a "breaking change"?

Reply:
Is this change compatible with both 44 and 45?  If so, I'd like to move this to a separate PR we merge to main before 45.